virt/virt-4-10-release-notes.adoc

```diff
@@ -77,6 +77,7 @@ The SVVP Certification applies to:
 
 //CNV-13680 Primary and secondary networks over a shared NIC on OVN clusters
 
+//CNV-15779 Network alerts when Kubemacpool detects duplicated MAC addresses
```

Comment:
@bgaydosrh Are you aware that there are multiple unrelated changes in this PR?

Reply:
Yes, Those changes are getting my version of the notes rebased with the current version. This happens many times with release note updates. And Peer review will check for this too.

I'll tag Oded with a release notes to get a QE resource, assuming you are signing off.

Reply:
I will create another PR for this change, come to think of it, as the overall RN ownership has been transferred to another writer for this release (previously it was me). Thanks for reminder.